docs/content/guides/recipes/howto-author-recipes/index.md

```diff
@@ -129,7 +129,7 @@ Now that your Recipe template has been stored, you can add it your Radius Enviro
 {{% codetab %}}
 
 ```bash
-rad recipe register myrecipe --environment myenv --resource-type Applications.Datastores/redisCaches --template-kind bicep --template-path myregistry.azurecr.io/recipes/myrecipe:1.1.0
+rad recipe register myrecipe --environment myenv --resource-type Applications.Datastores/redisCaches --template-kind bicep --template-path myregistry.ghcr.io/recipes/myrecipe:1.1.0
```

Comment:
Need help checking that the target path for the command would still technically be valid here or if this requires `azurecr`.

Reply:
```suggestion
rad recipe register myrecipe --environment myenv --resource-type Applications.Datastores/redisCaches --template-kind bicep --template-path ghcr.io/USERNAME/recipes/myrecipe:1.1.0
```